### DISCUSSION TITLE
Is it still maintained?

### DISCUSSION THREAD

**Discussion by asiriak** [reactiveui/refit#1461] [Q&A] [answered]
The last commit was on February 8th, 2022, and .NET 7 is not supported yet. That makes me question if this library is still maintained.

**Comment by siriak** [comment]
It's not https://github.com/reactiveui/refit/discussions/1447

**Comment by squadwuschel** [comment]
Is there an alternative to Refit?

**Reply by siriak** [reply]
@squadwuschel none that I'm aware of. So far, Refit is still working fine. Maybe there'll appear some alternatives once it stops working with the latest .NET or becomes hopelessly outdated.

**Reply by PopeAndrewJ** [reply]
Take a look at RestEase

**Comment by glennawatson** [comment, marked as answer]
New version was released yesterday.

**Reply by siriak** [reply]
Are you the maintainer for the foreseeable future?

**Reply by ChrisPulman** [reply]
Glenn and I will certainly be maintaining this library, we will do our best to provide the necessary support and updates going forward.
Understandably there's been a long time since anything has been done here but we will do our best to get things back on track.

Thanks to everyone for their patience.
